search/query/conjunction.go

```diff
@@ -26,6 +26,7 @@ import (
 type ConjunctionQuery struct {
 	Conjuncts       []Query `json:"conjuncts"`
 	BoostVal        *Boost  `json:"boost,omitempty"`
+	ArrayDepth      int     `json:"arraydepth,omitempty"`
```

Comment:
Per our JSON convention .. shouldn't this be array_depth?

Reply:
right. done!